examples/ios/webview/exampleApp/RecaptchaBridge.swift

```diff
@@ -0,0 +1,73 @@
+import Foundation
+import WebKit
+import RecaptchaEnterprise
+
+class RecaptchaBridge: NSObject, WKScriptMessageHandler {
+  
+  private var webView: WKWebView?
+
+  func userContentController(
+    _ userContentController: WKUserContentController, didReceive message: WKScriptMessage
+  ) {
+    guard let messageHandler = HandlerNames(rawValue: message.name) else {
+      return
+    }
+    
+    guard let messageBody = message.body as? [String: Any] else {
+      return
+    }
+    
+    switch messageHandler {
+    case .onExecute: onExecute(messageBody)
+    }
+  }
+  
+  enum HandlerNames: String, CaseIterable {
+    case onExecute = "onExecute"
+  }
+  
+  private func removeFromWebView(_ webView: WKWebView) {
+    self.webView = nil
+
+    for name in HandlerNames.allCases {
+      webView.configuration.userContentController.removeScriptMessageHandler(forName: name.rawValue)
+    }
+  }
+  
+  func addToWebView(_ webView: WKWebView) {
+    removeFromWebView(webView)
+    self.webView = webView
+    
+    for name in HandlerNames.allCases {
+      webView.configuration.userContentController.add(self, name: name.rawValue)
+    }
+  }
+  
+  private func onExecute(_ messageBody: [String: Any]) {
+    guard let action = messageBody["action"] as? String,
+          let bridgeResolve = messageBody["bridgeResolve"] as? String,
+          let bridgeReject = messageBody["bridgeReject"] as? String
+    else {
+      return
+    }
+    print (self.webView?.debugDescription ?? "webView is nil")
+    Task.detached(priority: .background){
```

Comment:
By using Task we are basically putting support only for iOS 13+. I'm not against it but are we sure we want that? Our current SDK has support for iOS 12+.

Reply:
based on offline discussion we settled on ios13 support (other changes were made to support that)